docs/changes.rst

```diff
@@ -5,12 +5,12 @@ Not yet released.
 
 * Improved user pages and added listing of users.
 * Dropped support for migrating from 3.x releases, migrate through 4.1 or 4.0.
-* Added exports in several monolingual formats.
+* Added exports for several monolingual formats.
```

Comment:
```suggestion
* Added exports into several monolingual formats.
```

Reply:
What are "exports" here?

Reply:
Downloading translations in other format, see https://docs.weblate.org/en/latest/user/files.html. The list is now extended to include several monolingual formats (Android, JSON, iOS,...).